src/interface/src/app/plan/project-areas/project-areas.component.html

```diff
@@ -31,10 +31,8 @@ <h3 class="title">Project Areas</h3>
     </tr>
   </thead>
   <tbody>
-    <tr
-      *ngFor="let area of areas"
-      [style.background-color]="colors[area.id - 1] + '4D'">
-      <td class="area-id">
+    <tr *ngFor="let area of areas">
+      <td class="area-id" [style.border-left-color]="colors[area.id - 1]">
```

Comment:
I should have asked this before, but is there any need to bounds check this?

Reply:
Good call on this! Ive updated to use the function `getColorForProjectPosition` which cycles thru the colors, and added tests and edge cases. Thanks!